src/main/scala/org/camunda/feel/impl/builtin/ConversionBuiltinFunctions.scala

```diff
@@ -170,16 +170,25 @@ class ConversionBuiltinFunctions(valueMapper: ValueMapper) {
     builtinFunction(
       params = List("from"),
       invoke = { case List(ValString(from)) =>
-        ValNumber(from)
+        parseNumber(from)
       }
     )
 
+  private def parseNumber(from: String): Val = {
```

Comment:
❓ Is it possible to move this parsing as built in behaviour for `ValNumber` ?

Reply:
Yes, this would be possible. As part of #65, I plan to remove the implicit conventions from the `package` object and add construction methods to the value types. 

However, this will be part of a bigger refactoring. :telescope: